subsys/bluetooth/controller/ll_sw/ull_iso_internal.h

```diff
@@ -4,21 +4,43 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
 /* Calculate ISO PDU buffers required considering SDU fragmentation */
-/* FIXME: Calculation considering both Connected and Broadcast ISO PDU
- *        fragmentation.
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+/* Internal ISO Tx SDU maximum length.
+ * A length that is minimum of the resultant combination of the HCI ISO data fragments provided and
+ * the user configured maximum transmit SDU length (CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX).
+ */
+#define BT_CTLR_ISO_TX_SDU_LEN_MAX MIN(((CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE * \
+					 CONFIG_BT_CTLR_ISO_TX_BUFFERS) - \
+					BT_HCI_ISO_SDU_HDR_SIZE), \
```

Comment:
I assume the point of this is optimization, i.e. if 

`CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=100`
`CONFIG_BT_CTLR_ISO_TX_BUFFERS=6`
and the application knows that it only ever sends SDUs of max size 200, then it can set
`CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=200`
so that `BT_CTLR_ISO_TX_SDU_LEN_MAX` becomes 200 instead of 600, right?

Reply:
yes.